libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -861,15 +861,16 @@ - (NSString *)scopeQueryParamString {
 - (void) updateCredentials:(NSDictionary *) params {
 
     // Logging event for token refresh flow.
+    SFUserAccount *userAccount = [[SFUserAccountManager sharedInstance] accountForCredentials:self.credentials];
```

Comment:
This is violating class hierarchy/visibility.  `SFOAuthCoordinator` needs no knowledge of user accounts, and we shouldn't add it simply for the sake of a shortcoming of our analytics API.  If we want to add an event here that e.g. SFUserAccountManager could subscribe to and take this action, I'm okay with that.  But this needs to be reconsidered.